cypress/integration/channel/channel_settings_spec.js

```diff
@@ -0,0 +1,42 @@
+
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint-disable max-nested-callbacks */
+describe('Channel Settings', () => {
+    before(() => {
+        // 1. Go to Main Channel View with "user-1"
+        cy.toMainChannelView('user-1');
+    });
+
+    it('C15052 All channel types have appropriate close button', () => {
+        cy.get('#publicChannel').parent().parent().children('li:not(:first-child)').not('#moreButton').each(($el) => {
+            cy.wrap($el).find('a').find('span.btn-close').should('have.length', 0);
+        });
+
+        cy.get('#privateChannel').parent().parent().children('li:not(:first-child)').not('#moreButton').each(($el) => {
+            cy.wrap($el).find('a').find('span.btn-close').should('have.length', 0);
+        });
+
+        // add a direct message incase there is not one
+        cy.get('#directChannel button').click();
+        cy.get('.more-modal__row.clickable').first().click();
+        cy.get('#saveItems').click();
+
+        cy.get('#directChannel').parent().parent().children('li:not(:first-child)').not('#moreButton').each(($el) => {
+            cy.wrap($el).find('a').find('span.btn-close').should('have.length', 1);
```

Comment:
In case there are more than one DMs where other DM has unread and without close button, this test fail.  I'd suggest to test condition for it.

Reply:
Thanks @saturninoabril I've added a step to click on all the DMs first. Sorry for the long delay on this, I just moved and have now settled in to a new place.